src/test/scala/nl.knaw.dans.easy.bagstore/component/BagStoresSpec.scala

```diff
@@ -99,6 +99,37 @@ class BagStoresSpec extends TestSupportFixture
     }
   }
 
+  it should "result in a failure when a get is done on a hidden file with forceInactive=false" in {
```

Comment:
We are talking about hidden bags, not a bout hidden files in bags? This phrasing confuses me.

Reply:
changed to hidden bags